marklogic-client-api/src/main/java/com/marklogic/client/datamovement/impl/QueryBatcherImpl.java

```diff
@@ -704,7 +703,6 @@ private class QueryTask implements Runnable {
       this.filtered = filtered;
       this.forestBatchNum = forestBatchNum;
       this.start = start;
-      this.isQueryBatch = isQueryBatch;
```

Comment:
Removed, `isQueryBatch` is not used anywhere and this was assigning the variable to itself.